entries.go

```diff
@@ -157,6 +157,52 @@ func GetAllEntries() ([]Item, error) {
 	return allEntries, err
 }
 
+// GetAllEntriesFiltered calls GetEntries with the given archive and starred parameters parameters, thus returning all articles as []wallabago.Item which match the filter
+func GetAllEntriesFiltered(archive, starred *bool) ([]Item, error) {
+	archiveInt := -1
+	if archive != nil {
+		switch *archive {
+		case true:
+			archiveInt = 1
+		case false:
+			archiveInt = 0
+		}
+	}
+	starredInt := -1
+	if starred != nil {
+		switch *starred {
```

Comment:
duplicate code should be avoided. analog to the archive lines right above.

Reply:
I removed that function all together.